lms/envs/common.py

```diff
@@ -5455,3 +5455,26 @@ def _should_send_certificate_events(settings):
 derived_collection_entry('EVENT_BUS_PRODUCER_CONFIG', 'org.openedx.learning.certificate.revoked.v1',
                          'learning-certificate-lifecycle', 'enabled')
 BEAMER_PRODUCT_ID = ""
+
+#### Survey Report ####
+# .. setting_name: ENABLE_SURVEY_REPORT
+# .. setting_default: True
+# .. setting_description: Set to True to enable the feature to generate and send survey reports.
+ENABLE_SURVEY_REPORT = os.environ.get('ENABLE_SURVEY_REPORT', True)
```

Comment:
I'm unfamiliar with this pattern of using `os.environ.get` in the Django settings file. What's it used for here?

Reply:
Hi @timmc-edx I had been using it because I saw it in another settings like: ENABLE_AUTHN_MICROFRONTEND, but I notice they are doing override of the settings using config file in this line: https://github.com/openedx/edx-platform/blob/5c58eb47bd06c0928b5890705be021754422ef51/lms/envs/production.py#L94 so I just set default values, thanks.